company.el

```diff
@@ -2909,10 +2916,10 @@ Delay is determined by `company-tooltip-idle-delay'."
 
 (defvar-local company-preview-overlay nil)
 
-(defun company-preview-show-at-point (pos)
+(defun company-preview-show-at-point (pos &optional common)
```

Comment:
I guessed wrong. The code seems not to be simplified. I will let the function to take the common and the completed strings will be selected by the number of argumets.